test/e2e/currentSpecReporter.js

```diff
@@ -0,0 +1,16 @@
+let currentSpec = undefined
+
+class CurrentSpecReporter {
```

Comment:
Would it be possible to type that file to express the fact that it implements the `jasmine.CustomReporter`?

Reply:
That would require to type the configuration which probably don't worth the trouble.
I'll go with https://github.com/DataDog/browser-sdk/pull/367#discussion_r411231844 to be cleaner anyway